### ISSUE TITLE
Correct Integer Division

### ISSUE THREAD

**Issue by acdelgado1019** [Wildcards-7506/FRC-2023#25]
SparkMaxes are generating errors due to a type mismatch when dividing numbers.
TLDR: Change any division in the across the robot from the form 1/X to 1.0/X

**FYE**
Code only does what you tell it to. When it sees a phrase like "1/9", it sees two integers being divided.
Integer division in code means dividing the two numbers and cutting off the decimal place.
When evaluating 1/9, which results in 0.11111, it makes sense that integer division would result in zero. 
This zero is sent to the Sparkmax, which doesn't know how to tell the encoder that it is not moving while it is moving (0 distance traveled per rotation).
In order to fix this, we need to pass a double into the equation (1.0/9 or 1.0/9.0). With the decimal place detected, the correct answer of 0.111111 will be generated.
